initial-test-data.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 # ==========================================================================
-# Initialization script for a wagtail user and imported data
+# Initialization script for a wagtail user and imported data.
```

Comment:
Should this ` Initialization script for a wagtail user and imported test data.`?